ui/perfherder/alerts/Assignee.jsx

```diff
@@ -43,11 +45,13 @@ export default class Assignee extends React.Component {
   };
 
   pressedEnter = async (event) => {
-    event.preventDefault();
     if (event.key === 'Enter') {
+      event.preventDefault();
```

Comment:
Could you quickly explain why did you move it?

Reply:
I moved this line (event.preventDefault();) inside the if statement because it was also preventing the onchange event on the input field, and that's why the input field was not editable. However, we still need that line because inside the if statement, 'const { failureStatus } = await updateAssignee(newAssigneeUsername);' is called. This line at the end of the function will try to redirect you to '/alerts' regardless of whether the assignment was successful or not.

Reply:
Thanks for the explanation